CHANGELOG.md

```diff
@@ -43,3 +49,9 @@
 > Mar 28, 2017
 
 * :nut_and_bolt: **New** Created `Collapse` component
+
+## Examples
```

Comment:
Maybe this part should be in a `CONTRIBUTE.md` document instead ?

Reply:
I think it is nice to have it here so that you can copy paste it easily :)

Reply:
Dont disagreed on that one, but we should probably create a contribute document anyways :P Lets make it an registered issue